mondey_backend/src/mondey_backend/databases/milestones.py

```diff
@@ -22,3 +24,94 @@ def create_database():
             session.add(Language(id="de"))
             session.add(Language(id="en"))
             session.commit()
+
+        if session.exec(select(ChildQuestion)).first() is None:
+            session.add(
+                ChildQuestion(
+                    id=1,
+                    order=0,
+                    component="textarea",
+                    type="text",
+                    text={
+                        "de": ChildQuestionText(
+                            child_question_id=1,
+                            lang_id="de",
+                            question="Name des Kindes?",
+                            options="",
+                        ),
+                        "en": ChildQuestionText(
+                            child_question_id=1,
+                            lang_id="en",
+                            question="Child'Name?",
+                            options="",
+                        ),
+                    },
+                )
+            )
+            session.add(
```

Comment:
the problem is here that the data of the child gets sourced through the same interface, i.e., `ChildDataInput.svelte`. For the user, we don´t have this because the user needs credentials upon registration and the questions then are a whole separate thing. So, unless we want to do something similar for the child (which from a user perspective would be a bit weird imho, but tolerable), we have to put entries for name and birth month/year/whatever into the questions table, and they need to be default questions because any new child MUST have them. The full date comes in b/c I think it's a more usable component in the frontend to have the usual calender selection widget rather than having to type out month and year.

Reply:
ok, so after our discussion, I removed the default questions. I believe that we would need some input to know what would be reasonable defaults, and the 'remark' question that was left after removing the ones that populate `Child` objects didn't seem useful enough to put in as a fixed thing.